project/project.sbt

```diff
@@ -2,7 +2,7 @@ import java.util.Locale
 
 libraryDependencies += "org.scala-sbt" % "scripted-plugin" % sbtVersion.value
 
-addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.0")
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.6")
 addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
```

Comment:
Sorry, this project is too meta. It is a library dependency of the plugin, but this plugin itself is not (for now) going to be used with sbt 1.0, it's simply going to be used by `sbt-web` etc to supply common settings for build themselves. So, the plugin it provides needs to provide the right cross build settings, but the plugin itself is not cross built, and so its dependencies also don't need to be cross built.